packages/cli/src/commands/build.ts

```diff
@@ -1,21 +1,31 @@
+
+import {Command, flags} from "@oclif/command"
+import {PromptAbortedError} from "../errors/prompt-aborted"
 import {build as ServerBuild} from "@blitzjs/server"
-import {Command} from "@oclif/command"
 
 export class Build extends Command {
-  static description = "Create a production build"
+  static description = "Creates a production build"
   static aliases = ["b"]
 
+  static flags = {
+    help: flags.help({char: "h"}),
+  }
+
   async run() {
     const config = {
       rootFolder: process.cwd(),
     }
 
     try {
+      // @ts-ignore noUnusedLocals
+      // This is needed to make sure that help flag is working correctly
+      const {flags}: {flags: Flags} = this.parse(Build)
```

Comment:
You can clean this up to be just `this.parse(Build)`. And you can move it up outside the try catch.

Reply:
Done 👍